source/javascripts/services/logger-service-instance.js.erb

```diff
@@ -0,0 +1,10 @@
+import { datadogLogs } from "@datadog/browser-logs";
+import DataDogLoggerService from "./logger-service";
+
+var loggerService = new DataDogLoggerService(datadogLogs, "<%= data[:constants][:datadog][:apiKey] %>");
```

Comment:
what if we say `global.logger = new DataDogLoggerService(...) ... factory(function { return global.logger; })`. So we have logger both on global and angular scope. That's nasty but at least we can mock it via karma by `global.logger = new Mock()` WDYT? Or we can also not test the logger itself, the logic is pure simply in it anyway.

Reply:
That's a good idea, I don't think it's that nasty, having it tested is a good trade-off. I'll try it.